src/redux/persist-helper.ts

```diff
@@ -18,10 +18,8 @@ export function getRehydratePayload(action: RehydrateAction, key: string) {
 
 // Note: Shouldn't be necessary except for in one place in the root saga
 export function* waitForRehydrate() {
```

Comment:
w00t w00t, thanks so much for looking into this!! ❤️ 

Since this helper is used in a single place and not supposed to be used anywhere else, 
what do you think of moving it to the root saga where it is used and removing the export?
This way we don't even need the global variable and this workaround.

Reply:
thanks for the suggestion! i like it, it does simplify things a lot. i guess it was originally like this to keep all the persist logic together, but i think either way a 1 liner in the root saga to wait for the rehydrate is a little easy to miss so i like this simplified version much better!